.github/workflows/prod-app-2-cd.yml

```
@@ -0,0 +1,121 @@
+name: Prod App 2 CD
+on:
+  workflow_call:
+    inputs:
+      run_id:
+        required: true
+        type: string
+    secrets: inherit
```

actionlint (1.7.7)

"secrets" section is scalar node but mapping node is expected

(syntax-check)